mocks/Client.go

```diff
@@ -0,0 +1,491 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
We should avoid linking this package/file into normal binaries, so it's best to rename this file to `client_test.go`. I would also move this file into the root dir and make it part of the `client` package, since this package does not have it's own purpose and is only used by test functions.